packages/internal/src/generate/graphqlCodeGen.ts

```diff
@@ -124,7 +125,7 @@ function getPluginConfig() {
     const localPrisma = require('@prisma/client')
     prismaModels = localPrisma.ModelName
     Object.keys(prismaModels).forEach((key) => {
-      prismaModels[key] = `.prisma/client#${key} as Prisma${key}`
+      prismaModels[key] = `@prisma/client#${key} as Prisma${key}`
```

Comment:
Both work, because prisma client is actually aliased to ./prisma in node_modules I think (that's where the generated types go) 

I just thought this was better, because it respects the yarn/node aliasing rather than overlook it.